holoviews/plotting/bokeh/chart.py

```diff
@@ -400,7 +400,7 @@ class HistogramPlot(ColorbarPlot):
 
     _nonvectorized_styles = ['line_dash', 'visible']
 
-    selection_display = BokehOverlaySelectionDisplay()
+    selection_display = BokehOverlaySelectionDisplay(color_prop=['color', 'fill_color'])
```

Comment:
Is this where we'd try varying alpha?

Reply:
Yeah, I think we'd need to list out all of the alpha properties that need to be overridden here

Reply:
@jbednar What was the discussion about alpha about? Is it something I should tackle now?

Reply:
@philippjfr , the discussion of alpha is [above](https://github.com/holoviz/holoviews/pull/4119#issuecomment-570268274), but I'll summarize the status as I know it. The  underlying idea here is that we want to "mute" the unselected glphs so that they fade more into the background, letting people focus on the selected ones.  Right now, link_selections is doing that by darkening the unselected color, which I believe is appropriate only for dark backgrounds (making unselected items more similar to the background); it should do the opposite for light backgrounds. But even if we had a switch in link_selections between darkening and lightening, it wouldn't work for the glaciers example, where the map plot has a dark background and the rest are light. Hence my proposal to decrease the opacity of unselected glyphs instead of changing their color. I believe that the result would be to make the unselected glyphs fade into the background in all cases, but I haven't tried it. To try it, you'd look at the various places where "alpha" is mentioned in the comments on this PR.